Makefile

```diff
@@ -112,15 +112,21 @@ gocyclo:
 pretest: check-fmt vet errcheck staticcheck
 
 test:
-	echo "" > coverage.txt
-	for pkg in $(PKGS);	do \
-		go test -v -tags unittest -coverprofile=profile.out -covermode=atomic $(BUILD_OPTIONS) $${pkg} || exit 1; \
-		if [ -f profile.out ]; then \
-			cat profile.out >> coverage.txt; \
-			rm profile.out; \
-		fi; \
-	done
-
+	  docker run --rm $(SECCOMP_OPTIONS) -v $(shell pwd):/go/src/github.com/libopenstorage/stork  $(DOCK_BUILD_CNT) \
+		      /bin/bash -c 'cd /go/src/github.com/libopenstorage/stork; \
+              PKGS=$$(go list ./... 2>&1 | grep -v "github.com/libopenstorage/stork/vendor" | grep -v "pkg/client/informers/externalversions" | grep -v versioned | grep -v "pkg/apis/stork" | grep -v "hack"); \
```

Comment:
instead of copy and paste PKGS variable, just run it inside a docker container in line 25:
```
ifndef PKGS
PKGS := $(shell docker run --rm $(SECCOMP_OPTIONS) -v $(shell pwd):/go/src/github.com/libopenstorage/stork  $(DOCK_BUILD_CNT) \
/bin/bash -c "cd /go/src/github.com/libopenstorage/stork && go list -buildvcs=false ./... 2>&1 | \
grep -v 'github.com/libopenstorage/stork/vendor' | \
grep -v 'pkg/client/informers/externalversions' | \
grep -v versioned | \
grep -v 'pkg/apis/stork' | \
grep -v 'hack'")
endif
```